CHANGES.rst

```diff
@@ -9,7 +9,7 @@ There's a frood who really knows where his towel is.
 - Move ``View`` class to the ``browser`` module.
   [hvelarde]
 
-- Remove unused ``poll_portlet_render`` view.
+- Remove unused ``poll_portlet_render`` view and empty ``poll.xml`` model.
```

Comment:
I just see the ``poll.xml`` removal here, not the poll portlet render view

Reply:
it's here: https://github.com/collective/collective.polls/pull/125/commits/0546f78c1c01c8e6a4615ea0b27192205a8c6352